README.md

```diff
@@ -82,8 +82,11 @@ To modify download information or change packaging information send a pull reque
 
 Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
 
+Be aware that if you build this software without removing the provided API keys in [CMakeLists.txt](CMakeLists.txt) you are accepting the following terms and conditions:
+ - [Microsoft Identity Platform Terms of Use](https://docs.microsoft.com/en-us/legal/microsoft-identity-platform/terms-of-use)
+ - [CurseForge 3rd Party API Terms and Conditions](https://support.curseforge.com/en/support/solutions/articles/9000207405-curse-forge-3rd-party-api-terms-and-conditions)
+If you do not agree with these terms and conditions, then remove the associated API keys from the [CMakeLists.txt](CMakeLists.txt) file.
```

Comment:
remove "or change" would be nicer

Reply:
Doesn't that imply that someone who does not agree with the terms and conditions can just change the key? They should still probably be agreeing to the terms if they use any key (although, of course, if they change the key then it becomes their responsibility, and it goes to Microsoft/CurseForge to enforce that, not PolyMC).

Reply:
Yea, I meant what you wrote in parens. That kind of clarification would be nice instead of a flat "then remove".

Reply:
I think it is better like this. We would be implying that it's fine to break ToS as long as they change the key, which it isn't.

Reply:
Hmm. Okay with me in that case.